### PR TITLE
Allow schemas to be written in YAML format as well

### PR DESCRIPTION
**Pull request by d00rman**
Writing JSON by hand is a painful experience. YAML is much better for that. Hence, this PR allows schema definitions to be written in YAML as well. The code first checks for the JSON definition and uses YAML as a fall-back.

### CODE REVIEW

**Comment by d00rman** [comment]
cc @wikimedia/services

File: lib/schema.js [hunk 1]
```diff
@@ -5,14 +5,16 @@ var ajv = require('ajv')();
...
 
 
 /** @const */
 var SCHEMA_REPO = path.join(__dirname, '../schemas');
 
 
-function schemaPath(schema) {
-    return path.join(SCHEMA_REPO, schema.toLowerCase() + '.js');
+function schemaPath(schema, ext) {
+    ext = ext || 'js';
```
Comment (gwicke) [inline lib/schema.js]: Shouldn't this be `json`?

Comment (d00rman) [inline lib/schema.js]: I'd say so as well, but it seems we are using [the `js` extension](https://github.com/wikimedia/restevent/tree/8873af9125df71b2d99335b87e748a3797063aa9/schemas)

Comment (gwicke) [inline lib/schema.js]: I'd be in favor of changing this while it's still easy. @eevans, are you okay with s/.js/.json/ ?

Comment (eevans) [inline lib/schema.js]: > I'd be in favor of changing this while it's still easy. @eevans, are you okay with s/.js/.json/ ?

+1

**Comment by d00rman** [comment]
`s/js/json` done

**Pull request merged**